README.md

```diff
@@ -16,40 +16,94 @@ Some of these are tricky!  Don't trust your first instinct.
 
 
 **a)** `"1" == 1`
+<pre>
+True. ==  tries to match types 
+</pre>
 
-**b)** `"1" === 1`
+**b)** `"1" === 1` 
+<pre> 
+False, not an exact match.
+</pre>
 
 **c)** `x == 'x'`
+<pre> 
+True, types converted. 
+</pre>
 
 **d)** `x == (x+'')`
+<pre> 
+True, for numbers, I feel like it should be true in general, buts its not! 
+</pre>
 
 **e)** `'' == ' '`
+<pre> 
+False, that is an empty string and not equal to anything. 
+</pre>
 
 **f)** `x = true`
 
+<pre> 
+that is an assignment operator, so x is equal to true now. 
+</pre>
 **g)** `var x; x == 'undefined'`
 
+<pre> 
+with no definition, x is technically undefined, the number, not the string. 
+</pre>
 **h)** `'9'<'10'`
 
+<pre> 
+True
+</pre>
 **i)** `typeof x + 1 === "number"`
 
+<pre>
+this will always includde a string value, so it is false. 
+</pre>
 **j)** `typeof x % 2 === "number"`
 
+<pre> 
+false in all cases, since a % operator is invalid on a non-number, and will throw a NaN. 
+</pre>
 **k)** `typeof (x % 2) === "number"`
+<pre> 
+True, if x is a number. False otherwise.
```

Comment:
This will actually be true for all values, as the result will be either a number or `NaN`, and `NaN` is (strangely) a number.